### PR TITLE
Fix preview in Spectral Deconvolution (MCR and Hierarchical Clustering)

### PR DESCRIPTION
**Pull request by asmirn1**

### CODE REVIEW

File: src/main/java/io/github/mzmine/modules/dataprocessing/adap_hierarchicalclustering/EICPlot.java [hunk 1]
```diff
@@ -145,10 +145,19 @@ public void updateData(List<List<NavigableMap<Double, Double>>> clusters, List<D
...
           series.add(e.getKey(), e.getValue());
 
-        xyDataset.addSeries(series);
-        colorDataset.add(color);
-        toolTips.add(info.get(i).get(j));
+        try{
+          xyDataset.addSeries(series);
+        }
+        catch(Exception e) {
+          System.out.println(e.getMessage());
```
Comment (robinschmid) [inline src/main/java/io/github/mzmine/modules/dataprocessing/adap_hierarchicalclustering/EICPlot.java:152]: use logger.log(Level.WARNING, e.getMassage(), e);

Comment (robinschmid) [reply src/main/java/io/github/mzmine/modules/dataprocessing/adap_hierarchicalclustering/EICPlot.java:152]: why would this trow an exception though?

**Review by robinschmid — COMMENTED** [review]
Tiny comments. Otherwise it looks good. But I had no time to test it.

**Comment by asmirn1** [comment]
@varunotelli Could you take a look at Robin's comments?

**Pull request merged**